BodyText/tests/BodyText-specs.js

```diff
@@ -1,39 +1,75 @@
-import {mount} from 'enzyme';
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+
 import BodyText from '../BodyText';
-import css from '../BodyText.module.less';
 
 describe('BodyText Specs', () => {
+	test('should render a text', () => {
+		render(
+			<BodyText>Hello! </BodyText>
+		);
```

Comment:
let's have this on 1 line, on all tests, except for the ones which have multiple components or multiple-line text

Reply:
Done